sale_channel_search_engine/models/sale_channel.py

```diff
@@ -9,3 +9,18 @@ class SaleChannel(models.Model):
     _inherit = "sale.channel"
 
     search_engine_id = fields.Many2one("se.backend", "Search Engine")
+
+    def open_se_binding(self):
+        action = self.env.ref("connector_search_engine.se_binding_action").read()[0]
+        indexes = self.search_engine_id.index_ids.ids
+
+        action["domain"] = [("index_id", "in", indexes)]
+        action["context"] = "{'hide_res_model': True}"
+        return action
+
+    def synchronize_channel_index_action(self):
+        self._synchronize_channel_index()
+        return self.open_se_binding()
+
+    def _synchronize_channel_index(self):
+        pass
```

Comment:
Not sure that we need this (for the product sync is immediatly, and for the category it's depend on the implementation). To discuss